package.json

```diff
@@ -5,20 +5,20 @@
   "main": "index.js",
   "dependencies": {
     "concat-stream": "~1.6.0",
-    "duplexer2": "~0.0.2",
-    "escodegen": "~1.3.2",
+    "duplexer2": "0.1.4",
```

Comment:
Can you keep the `~` prefix for all versions? Otherwise people pulling this in won't be able to get patch updates automatically.